crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -587,13 +591,75 @@ mod private {
             Ok(Self { room, store, events, waited_for_initial_prev_token: false })
         }
 
+        /// Removes the bundled relations from an event, if they were present.
+        ///
+        /// Only replaces the present if it contained bundled relations.
+        ///
+        /// Returns an `Option` so as to be able to short-circuit with `?`, but
+        /// the result is meaningless and mustn't be used.
```

Comment:
I think I'd prefer to avoid including this `Option` in the type signature, perhaps by using an inner function here, but I'm not going to insist on it.

If you leave it as-is, please can you explicitly say that it always returns `None`? That would help me understand its (lack of) meaning.